backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/service/discord/DefaultDiscordUserVerificationService.java

```diff
@@ -119,58 +125,98 @@ public Either<Problem, DiscordStartVerificationResponse> startVerification(Strin
 
     @Override
     @Transactional
-    public Either<Problem, IsVerifiedResponse> checkVerification(DiscordCheckVerificationRequest checkVerificationRequest) {
-        var eventId = checkVerificationRequest.getEventId();
-        var eventDetails = chainFollowerClient.findEventById(eventId);
+    public Either<Problem, IsVerifiedResponse> checkVerification(DiscordCheckVerificationRequest request) {
+        String eventId = request.getEventId();
+        String walletId = request.getWalletId();
+        Optional<WalletType> walletIdType = request.getWalletIdType();
 
-        if (eventDetails.isEmpty()) {
-            log.error("event error:{}", eventDetails.getLeft());
+        if (!walletIdType.isPresent()) {
```

Comment:
I am confused about this, I thought Cardano WalletType is the default, yet we are demanding that this gets passed as a parameter. Actually in practise this can never be not present unless Lombok.Default annotation didn't work properly.

Reply:
You are right, this check is not necessary as `WalletType` is at least CARDANO.
I just removed it.